soybase/jekyll-soybase
# Makefile OS update and marker collection renaming

Update datastore-metadata submodule and tools/translate/index.html to reflect renaming of collection s/glyma.Wm82_ISU01.gnm2/glyma.Wm82.gnm6/, and update CommandLineTools to macosx15.0 in Makefile.

To test, first do
```
  git submodule update --remote _data/datastore-metadata
```
To test the updated track in jbrowse, do
```
  make jbrowse
```

Flagging @nathanweeks to check the Makefile, @jd-campbell to check the data update, and @ChenProm-USDA in case of conflicts with soybase-audit-2
